code/assurance/build.gradle

```diff
@@ -0,0 +1,257 @@
+/**
+ * Copyright 2022 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+plugins {
+    id 'com.android.library'
+    id 'jacoco'
+    id 'maven-publish'
+    id 'signing'
+    id 'com.diffplug.spotless'
+}
+
+// Apply local gradle tasks
+apply from: '../codeformat.gradle'
+
+jacoco {
+    toolVersion = rootProject.ext.jacocoVersion
+}
+
+if (project.hasProperty("signKeyLoc") && project.hasProperty("signKeyId") && project.hasProperty("signPass")) {
+    ext["signing.secretKeyRingFile"] = "$signKeyLoc"
+    ext["signing.keyId"] = "$signKeyId"
+    ext["signing.password"] = "$signPass"
+}
+
+if (project.hasProperty("sonatypeUser") && project.hasProperty("sonatypePass")) {
+    ext["ossrhUsername"] = "$sonatypeUser"
+    ext["ossrhPassword"] = "$sonatypePass"
+} else {
+    ext["ossrhUsername"] = ""
+    ext["ossrhPassword"] = ""
+}
+
+android {
+    compileSdkVersion rootProject.ext.compileSdkVersion
+
+    defaultConfig {
+        minSdkVersion rootProject.ext.minSdkVersion
+        targetSdkVersion rootProject.ext.targetSdkVersion
+        versionCode 1
```

Comment:
I think this gets embedded in the manifest. Can you remove versionCode and versionName?